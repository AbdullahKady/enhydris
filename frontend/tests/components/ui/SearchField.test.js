import { mount } from "@vue/test-utils";
import SearchField from "@/components/ui/SearchField";
import Vue from "vue";
import Buefy from "buefy";

Vue.use(Buefy);

describe("SearchField", () => {
  const wrapper = mount(SearchField, {
    mocks: {
      $t: x => x
    }
  });

  it("renders correctly", () => {
    expect(wrapper.element).toMatchSnapshot();
  });
});
