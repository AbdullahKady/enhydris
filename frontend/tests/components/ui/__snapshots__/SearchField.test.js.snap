// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchField renders correctly 1`] = `
<div>
  <div
    class="field"
  >
    <!---->
     
    <div
      class="control has-icons-left is-clearfix"
    >
      <input
        autocomplete="on"
        class="input"
        placeholder="search"
        type="text"
      />
       
      <span
        class="icon is-left"
      >
        <i
          class="mdi mdi-search mdi-24px"
        />
      </span>
       
      <!---->
       
      <!---->
    </div>
     
    <!---->
  </div>
</div>
`;
